control_plane/src/local_env.rs

```diff
@@ -39,6 +42,15 @@ pub struct LocalEnv {
     #[serde(with = "hex")]
     pub tenantid: ZTenantId,
 
+    // jwt auth token used for communication with pageserver
+    pub auth_token: String,
+
+    // used to determine which auth type is used
+    pub auth_type: AuthType,
```

Comment:
This way it allows soft launch of the feature and ease of local testing. You can have generated token in place but disabled auth, to enable auth you need to change only auth type. I agree with you that it will be clearer without condition checks. But token here is always Some, and there is only one check for LocalEnv [here](https://github.com/zenithdb/zenith/pull/394/files#diff-bb71ec79f533ac1e0735348d9ed68d4cd3a7e8d848064c47c817fc43563db7a5R34). Also there is an option to remove both this parameters in favor of using only connection string since token is just a password. What do you think?